src/libs/Clipboard/types.ts

```diff
@@ -0,0 +1,9 @@
+type SetString = (text: string) => void;
+
+type Clipboard = {
+    setString: SetString;
+    canSetHtml: () => void;
+    setHtml: (html: string, text: string) => void;
```

Comment:
Why not creating types for these functions and using it there the same way you did for `setString`?

Reply:
Done